tests/v1/lifecycle/test_app_streams.py

```diff
@@ -220,6 +219,77 @@ async def query_rbac_override():
     assert result.status_code == 200
 
 
+def test_get_relevant_app_stream_resource_definitions_with_ungrouped_permission(api_prefix, client):
+    """Testing a case with group None, which means 'ungrouped'"""
```

Comment:
This is an excellent example of when doc strings are helpful for tests. What you have is fine but another way to write this is with the template "Given [scenario] test/ensure/assert that [expected result]".